cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -307,8 +320,8 @@ public void init(@Nullable final NotificationReceivedListenerBase notificationRe
      * @param notificationReceivedListener callback for the notification received
      * @param notificationOpenedListener   callback for the notification opened
      */
-    public void init(String channelId, @Nullable final NotificationReceivedListenerBase notificationReceivedListener, @Nullable final NotificationOpenedListener notificationOpenedListener) {
-        init(channelId, notificationReceivedListener, notificationOpenedListener, null);
+    public void init(String channelId, @Nullable final NotificationReceivedListenerBase notificationReceivedListener, @Nullable final NotificationOpenedListener notificationOpenedListener, @Nullable final NotificationOpenedCallbackListener notificationOpenedCallbackListener) {
+        init(channelId, notificationReceivedListener, notificationOpenedListener, notificationOpenedCallbackListener, null);
```

Comment:
We do not want to change the signature of the init methods. The customer should be able to use NotificationOpenedListener OR NotificationOpenedCallback listener in the same argument

Reply:
Okay i will check and change it